service/job/job_type_controller.py

```diff
@@ -121,13 +120,226 @@ def get_job_type_detail(job_type):
                 ],
                 "title": "Ingest a Journal-Issue to iDAI.publications/journals"
             },
-            "params": {
-                "do_ocr": "boolean",
-                "files": "list",
-                "metadata": "dict",
-                "nlp_params": "dict",
-                "ojs_metadata": "dict",
-                "parts": "list"
+            "schema": {
```

Comment:
Können wir das Schema hier nicht etwas kürzen? Das ja schon echt ein riesiger docstring ...

Reply:
Vorschläge? Man will ja in den API-Doc schon ein vernünftiges Beispiel sehen oder? Ich könnte das Schema etwas einkürzen, aber dann wundert sich vielleicht später jemand und benutzt das falsch oder so.